src/test/resources/large/.gitattributes

```diff
@@ -9,3 +9,4 @@ human_g1k_v37.20.21.fasta.fai filter=lfs diff=lfs merge=lfs -text
 exampleLargeFile.txt filter=lfs diff=lfs merge=lfs -text
 dbsnp_138.b37.1.1-65M.vcf filter=lfs diff=lfs merge=lfs -text
 dbsnp_138.b37.1.1-65M.vcf.idx filter=lfs diff=lfs merge=lfs -text
+human_g1k_v37.20.21.2bit filter=lfs diff=lfs merge=lfs -text
```

Comment:
I used the `faToTwoBit` binary from the "kent" UCSC utilities package.  Added a note to the commit message.